twiml/voice/gather/gather-3/gather-3.5.x.cs

```diff
@@ -1,17 +1,19 @@
 using Twilio.TwiML;
+using Twilio.TwiML.Voice;
+using System;
 
 
 class Example
 {
     static void Main()
     {
         var response = new VoiceResponse();
-        var gather = new Gather(action: "/process_gather.php", method: "GET");
+        var gather = new Gather(action: new Uri("/process_gather.php"), method: "GET");
         gather
             .Say("Please enter your account number,\nfollowed by the pound sign");
         response.Gather(gather);
```

Comment:
@joliveros Also, this change should be `response.Append(gather)`

Reply:
good catch